pallets/acurast/src/attestation.rs

```diff
@@ -273,7 +273,7 @@ pub fn validate_certificate_chain_root(
 /// - the next certificate's public key signs the next one and so on...
 pub fn validate_certificate_chain<'a>(
```

Comment:
`validate_certificate_chain` seems to be called 3 times on every attestation submission. We should try to only validate the certificate once and reuse the results.